pallets/README.template.md

```diff
@@ -0,0 +1,49 @@
+# {Name} Pallet
+
+{Short description of the pallet}
+
+## Summary
+
+{What does this pallet do?}
+
+### {Key Concept}
+
+{Description}
+
+### Actions
+
+The {Name} pallet provides for:
+
+- {Feature}
+- {Feature}
+- {Feature}
+
+## Interactions
+
+### Extrinsics
+
+| Name/Description                 | Caller | Payment | Key Events                                                                                                    | Runtime Added |
+| -------------------------------- | ------ | ------- | ------------------------------------------------------------------------------------------------------------- | ------------- |
+| `{extrinsic}`<br />{Description} | Tokens | Free    | [`{Event}`](https://frequency-chain.github.io/frequency/{pallet_name}/pallet/enum.Event.html#variant.{Event}) | 1             |
+
+See [Rust Docs](https://frequency-chain.github.io/frequency/{pallet_name}/pallet/struct.Pallet.html) for more details.
+
+### State Queries
+
+| Name      | Description         | Query                    | Runtime Added |
+| --------- | ------------------- | ------------------------ | ------------- |
+| {Query 1} | {Query Desctiption} | `{queryCallInCamelCase}` | 1             |
```

Comment:
Desctiption -> Description

Reply:
this wasn't my change, sorry, it was from conflicting stuff from main.